app/policies/comment_policy.rb

```diff
@@ -0,0 +1,31 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :user, :comment
+  def initialize(user, comment)
+    @user = user
+    @comment = comment
+  end
+
+  def create?
+    true
+  end
+
+  def new?
+    create?
+  end
+
+  def edit?
+    update?
+  end
+
+  def update?
+    author?
+  end
+
+  def author?
+    user.id == @comment.author_id
```

Comment:
Can simplify this with `user == comment.author` I think. Also, put this in a `private` block

Reply:
done!